maat-court-data-api/src/test/java/gov/uk/courtdata/integration/contribution/ContributionsControllerIntegrationTest.java

```diff
@@ -180,4 +180,19 @@ void givenAValidRepIdAndEmptyCorrespondence_whenGetContributionCountIsInvoked_th
         assertTrue(runNotFoundErrorScenario("Contributions entry not found for repId " + INVALID_REP_ID,
                 get(ENDPOINT_URL + "/" + INVALID_REP_ID).contentType(MediaType.APPLICATION_JSON)));
     }
+
+    @Test
+    void givenValidRepId_whenGetContributionsSummaryIsInvoked_thenCorrectResponseIsReturned() throws Exception {
+        MvcResult result = runSuccessScenario(get(ENDPOINT_URL + "/" + TestModelDataBuilder.REP_ID
+                + "/summary").contentType(MediaType.APPLICATION_JSON));
+
+        Assertions.assertThat(result.getResponse().getContentAsString())
+                .contains("CONTRIBUTIONS_202307111234.xml");
```

Comment:
Can we assert all the field with objectMapper.writeValueAsString()?

Reply:
I tried this but the assertion fails as the object built from the projection has the fields in a different order than that returned in the response. I couldn't find a way to affect the field ordering for the projection so have left it as is. Though if you have know how to solve I'm all ears =)